### PR TITLE
fix(ui): career site fixes on position page

### LINKED ISSUE #24: Career site fixes: Position page
**Problem**
On Career site, Position page:

- Heading: incorrect font-size 20px
- Application form section:
  - Outdated heading text “Apply for this job”
  - Incorrect inputs size - Small
  - Button stretched to full-width on all resolutions

**Solution**
- Heading: increase font-size from 20px to 24px (shouldn’t be applied for Application form)
- Application form section:
  - Change heading from “Apply for this job” to “Apply for this position”
  - Change inputs size to Medium
  - Button should be stretched to full-width only on low resolutions (Small, Extra small breakpoints)

### PR DESCRIPTION
**Pull request by Lyams**
Closes #24.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/views/career_site/positions/show.html.slim [hunk 1]
```diff
@@ -9,7 +9,7 @@
 .row#position_heading.mt-3.gap-3.justify-content-between
   .col-12.col-md-10
     .col
-      h2 = @position.name
+      h1 = @position.name
```
Comment (wrmk) [inline app/views/career_site/positions/show.html.slim:12]: as i understand we should not make this change
"Heading: increase font-size from 20px to 24px (shouldn’t be applied for Application form)"

and as I can see for the index page it is already 24px

File: app/views/career_site/positions/show.html.slim [hunk 2]
```diff
@@ -47,13 +47,15 @@
...
           required: true,
+          size: :medium,
           placeholder: "Full name" \
         )
       = render TextInputComponent.new( \
           form,
           method: :email,
           required: true,
           type: :email,
+          size: :medium,
```
Comment (wrmk) [inline app/views/career_site/positions/show.html.slim:58]: not sure that input takes medium size, on lookbook it has different padding
we took styles from `form-control`
image

File: app/views/career_site/positions/show.html.slim (hunk 1 above)
Comment (Lyams) [reply app/views/career_site/positions/show.html.slim:12]: Issue talks only about show.
[MockUp](https://www.figma.com/design/kQOH4s5DnfWL0p6o6tlMQz/Draft-%5BATS%5D?node-id=1520-10620&node-type=canvas&t=JhKHthbxjOk1CNyI-0)

Comment (wrmk) [reply app/views/career_site/positions/show.html.slim:12]: ok, agree. seems "shouldn’t be applied for Application form" is for `Apply for this position` text

**Review by wrmk — APPROVED** [review]

**Pull request merged**